integration_tests/OAuth.md

```diff
@@ -0,0 +1,110 @@
+# Manually testing of OAuth authentication using KeyCloak
```

Comment:
How does this file refer to integration tests?

Reply:
As there is not a clear way how to automate the test for OAuth, the markdown file was added, describing the way to manually test the authentication.